py/jupyterlite/src/jupyterlite/addons/base.py

```diff
@@ -33,6 +33,11 @@ class BaseAddon(LoggingConfigurable):
 
     manager: LiteManager = Instance(LiteManager)
 
+    ignore_sys_prefix: bool = Bool(
+        False,
+        help="ignore components from sys.prefix",
+    ).tag(config=True)
```

Comment:
hm... i wonder if this should even be configurable... it will get clobbered by the manager.

Reply:
Right we can keep `LiteBuildConfig` as the only place to configure this.